src/renderer/services/validator/interop_keys.ts

```diff
@@ -0,0 +1,15 @@
+import {PrivateKey} from "@chainsafe/bls";
+import {toBufferBE} from "bigint-buffer";
+import {bytesToBigInt, intToBytes} from "@chainsafe/lodestar-utils";
+import {hash} from "@chainsafe/ssz";
+
+const CURVE_ORDER = BigInt("52435875175126190479447740508185965837690552500527637822603658699938581184513");
+
+export function getInteropKey(index: number): PrivateKey {
```

Comment:
What's this? Sounds like it should be a some part of lodestar

Reply:
It some convention to generate determinitic keys that lighthouse is using. Lodestar is using PrivateKey.fromInt() which is not the same